### PR TITLE
Fix Issues in the Installing Ballerina & Downloads Pages

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Fix issues in the Installing Ballerina & Downloads pages.
> Fixes https://github.com/ballerina-platform/ballerina-dev-website/issues/528 & https://github.com/ballerina-platform/ballerina-dev-website/issues/540

## Check List

- [ ] **Page Addition**
  - [ ] Add `permalink` to pages
  - [ ] If contains empty folder(s), Add front-matter `redirect_to:`

- [ ] **Page Rename**
  - [ ] Add front-matter `redirect_from`
  - [ ] Add front-matter `redirect_to:` (If applicable)

### CODE REVIEW

File: learn/installing-ballerina.md [hunk 1]
```diff
@@ -52,13 +49,13 @@ The installer should put the `C:\Program Files\Ballerina\<ballerina-directory>\b
...
+[Download the DEB file](/downloads) and double-click on it to launch the installer. The installer guides you through the installation process and installs the Ballerina distribution to `/usr/lib/ballerina`.
 
-* If you downloaded the DEB file, use the following command to install Ballerina. Replace `<ballerina-home>/<ballerina-binary>.deb` with the actual file path. This installs Ballerina to the `/usr/lib/ballerina` directory.
+> **Info:** Alternatively, you can use the below command to install Ballerina using the downloaded the DEB file. Replace `<ballerina-home>/<ballerina-binary>.deb` with the actual file path.
 
-  ```
-  dpkg -i <ballerina-binary>.deb
-  ```
+```
+dpkg -i <ballerina-binary>.deb
```
Comment (keizer619) [inline learn/installing-ballerina.md:57]: @praneesha Shall we include rpm installation as well?

Comment (praneesha) [reply learn/installing-ballerina.md:57]: @tharik - What's the command for the RPM installation? Also, we haven't given an RPM installation for the latest release.

Comment (praneesha) [reply learn/installing-ballerina.md:57]: Fixed the above.

File: 1.0/learn/installing-ballerina.md [hunk 2]
```diff
@@ -10,28 +10,25 @@ redirect_from:
...
-## Downloading the Ballerina distribution
+## Prerequisites
 
-[Download](https://ballerina.io/downloads/) a Ballerina distribution based on your operating system.
-
-> Ballerina binary distributions are available for the following supported operating systems and architectures. Ensure that your system meets the below requirements before you proceed with the installation.
+Ballerina binary distributions are available for the following supported operating systems and architectures. Ensure that your system meets the below requirements before you proceed with the installation.
 - Windows Vista SP2 x64 or later
 - Ubuntu Linux 12.04 x64 - LTS and above
 - OS X 10.8.3 x64 and above
```
Comment (pubudu91) [inline 1.0/learn/installing-ballerina.md:25]: I think we need to revisit this list of supported OS. Some concerns:
- Vista is no longer supported by Microsoft (ended in 2017 apparently).
- Ubuntu is not the only linux OS we support.

Comment (praneesha) [reply 1.0/learn/installing-ballerina.md:25]: @pubudu91 Can you create a new GitHub issue with the updated list that should be included?

Comment (pubudu91) [reply 1.0/learn/installing-ballerina.md:25]: will do

Comment (praneesha) [reply 1.0/learn/installing-ballerina.md:25]: Thanks a lot!

Comment (pubudu91) [reply 1.0/learn/installing-ballerina.md:25]: https://github.com/ballerina-platform/ballerina-dev-website/issues/591

**Pull request merged**